version.py

```diff
@@ -40,18 +40,27 @@
 
 
 def call_git_describe(prefix='', cwd='.'):
-    cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    version_name_cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    last_tag_cmd = 'git describe --tags --abbrev=0'
```

Comment:
Isn't `--match` necessary in both commands? Otherwise `last_tag_cmd` can be a tag that does not have the prefix. Below you're assuming that it must have the prefix because you're stripping it.

Since it's obviously your assumption that both `version` and `last_tag` start with the prefix and the code is broken if they don't maybe it would be good idea to have assertions against that. If you had these asserts they would catch this problem.

Reply:
Done